### PR TITLE
Epmlstrtai 42. Done SuitController and SuitService.

### PR DESCRIPTION
**Pull request by FunCat**

### CODE REVIEW

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/dao/SuitDAOImpl.java [hunk 1]
```diff
@@ -4,30 +4,45 @@
 import com.epam.test_generator.entities.Suit;
 import org.springframework.stereotype.Repository;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @Repository
 public class SuitDAOImpl implements EntitiesDAO<Suit>{
```
Comment (EvilXa0S) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/dao/SuitDAOImpl.java]: I suggest to mark in the name of the class that it is temporary solution ie SuitDaoMockImlp, and then later autowire it with name and qualifier.

**Review by Walrus94 — APPROVED** [review]

File: Java/JDI/jdi-cucumber-test-generator/pom.xml [hunk 2]
```diff
@@ -66,7 +71,13 @@
         <dependency>
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
-            <version>3.1.0</version>
+            <version>${servlet-api.version}</version>
```
Comment (EvilXa0S) [inline Java/JDI/jdi-cucumber-test-generator/pom.xml]: Once again I suggest to introduce dependency management section in parent pom and store all out version there.

Comment (FunCat) [reply Java/JDI/jdi-cucumber-test-generator/pom.xml]: I added commit to the Epmlstrtai 40.

File: Java/pom.xml [hunk 3]
```diff
@@ -45,6 +45,8 @@
         <aspectj.version>1.8.3</aspectj.version>
         <project.build.sourceEncoding>utf8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>utf8</project.reporting.outputEncoding>
+        <fasterxml.version>2.5.1</fasterxml.version>
+        <servlet-api.version>3.1.0</servlet-api.version>
```
Comment (FunCat) [inline Java/pom.xml:49]: Add parent pom.xml versions.

**Pull request merged**